File: examples/fetch.py
# pip install requests — no key needed.
import requests
data = requests.get("https://projectcostestimator.com/api/cost-data").json()["data"]
base = data["base_project_types"]["landing_page"]["base_price_usd"]  # 300
agency = data["pricing_tier_multipliers"]["agency"]["global"]        # 2.2
print("Landing page (agency), base USD:", round(base * agency))

File: examples/fetch.js
// Node 18+ / browser — no key needed.
const res = await fetch('https://projectcostestimator.com/api/cost-data');
const { data } = await res.json();
const base = data.base_project_types.landing_page.base_price_usd; // 300
const geo  = data.geographic_multipliers.eastern_europe.price;    // 0.55
console.log('Landing page (EU freelancer), base USD:', Math.round(base * geo));
